multiaccounts/settings/database.go

```diff
@@ -369,7 +369,8 @@ func (db *Database) GetSettings() (Settings, error) {
 		waku_bloom_filter_mode, webview_allow_permission_requests, current_user_status, send_status_updates, gif_recents,
 		gif_favorites, opensea_enabled, last_backup, backup_enabled, telemetry_server_url, auto_message_enabled, gif_api_key,
 		test_networks_enabled, mutual_contact_enabled, profile_migration_needed, is_sepolia_enabled, wallet_token_preferences_group_by_community, url_unfurling_mode,
-                omit_transfers_history_scan, mnemonic_was_not_shown
+		omit_transfers_history_scan, mnemonic_was_not_shown, wallet_show_community_asset_when_sending_tokens, wallet_display_assets_below_balance,
```

Comment:
I see that the default values are set in the SQL script for these fields, but pretty sure, it's a good thing to have them added to the `CreateSettings` function as well, since that lets the status-go's client set them based on its need when initializing the app.